prometheus/aws-ecs/image/entrypoint.sh

```diff
@@ -11,10 +11,20 @@ if [[ -z $CORALOGIX_ENDPOINT ]]; then
   exit 1
 fi
 
+if [[ -z $SCRAPE_INTERVAL ]]; then
+  echo "SCRAPE_INTERVAL environment variable must be set"
+  exit 1
+fi
+
 DOCKERHOST=`cat /hostetc/hostname`
+CLUSTERNAME=`wget -qO- http://172.17.0.1:51678/v1/metadata | sed -e 's/[{}]/''/g' | awk -F , '{split($1,a,"\"");print a[4]}'`
```

Comment:
should verify that value is returned otherwise fail.

Reply:
added the variable check